bentoml/saved_bundle/templates.py

```diff
@@ -109,9 +109,9 @@
 # copy over model files
 COPY --chown=bentoml:bentoml . ./
 
-# the env var $PORT is required by heroku container runtime
-ENV PORT 5000
-EXPOSE $PORT
+# Default port for service
+ENV BENTOML_PORT 5000
+EXPOSE $BENTOML_PORT
```

Comment:
I think we also need `EXPOSE $PORT` here in order for it to work with Heroku right?  are you sure this change works when you also set the `port` to a different value on heroku?

Reply:
never mind, I think I misunderstood the `EXPOSE` command. Looking at related documentation here https://docs.docker.com/engine/reference/builder/#expose, I think it's ok if we just remove `ENV BENTOML_PORT 5000` here and change this line to `EXPOSE 5000`, that way we only define the default port once in the BentoML config, what do you think?

Reply:
I'm not sure. So one doubt is, suppose we use this with something like a docker-compose and override the BENTOML_PORT with  the environment variable to another port, should we also use EXPOSE on that port number or is EXPOSE suppose to be always a fixed port number?

Reply:
@jjmachan per the linked documentation, EXPOSE doesn't actually do anything: `The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the -p flag on docker run to publish and map one or more ports`

If the person who runs the container overrides it via  BENTOML_PORT env var, this person should also be aware to use `-p` to publish that port.  In other cases, `EXPOSE 5000` tells people that this is the default port that will be used.

Reply:
Thanks for explaining it @parano! I've made the changes